lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -349,23 +355,24 @@ export class ServerRequestParameters {
     }
 
     /**
-     * @hidden
-     *
-     * Validate  scopes/state/correlationId set in the request by the user
+     * generate unique state per request
      * @param request
      */
-    private setRequestServerParams(scopes: Array<string>, state: string, correlationId: string, clientId: string) {
-        // set scope to clientId if null
-        this.scopes = scopes? [ ...scopes] : [clientId];
-
+    static setState(state: string): string {
```

Comment:
should this be a static method on this instance class? 
This should probably be a util somewhere. 
Also unsure setState is the right name. 

This is something like `UrlUtils.getRequestState(userProvidedState: string): string `

Reply:
Done